Doc/library/asyncio-task.rst

```diff
@@ -602,6 +602,86 @@ Waiting Primitives
            # ...
 
 
+Running in Threads
+==================
+
+.. coroutinefunction:: to_thread(func, /, \*args, \*\*kwargs)
```

Comment:
You also need to add this function to the "high level APIs index"

Reply:
Will do, thanks for the reminder.

Reply:
Done.